androidtv/constants.py

```diff
@@ -310,88 +316,128 @@
 APP_FRANCE_TV = 'fr.francetv.pluzz'
 APP_GOOGLE_CAST = 'com.google.android.apps.mediashell'
 APP_GOOGLE_TV_LAUNCHER = 'com.google.android.apps.tv.launcherx'
+APP_HAYSTACK_NEWS = 'com.haystack.android'
 APP_HBO_GO = 'eu.hbogo.androidtv.production'
 APP_HBO_GO_2 = 'com.HBO'
 APP_HULU = 'com.hulu.plus'
 APP_IMDB_TV = 'com.amazon.imdb.tv.android.app'
+APP_IPTV = 'ru.iptvremote.android.iptv'
 APP_IPTV_SMARTERS_PRO = 'com.nst.iptvsmarterstvbox'
 APP_JELLYFIN_TV = 'org.jellyfin.androidtv'
 APP_KODI = 'org.xbmc.kodi'
+APP_LIVE_CHANNELS = 'com.google.android.tv'
+APP_MIJN_RADIO = 'org.samsonsen.nederlandse.radio.holland.nl'
 APP_MOLOTOV = 'tv.molotov.app'
+APP_MX_PLAYER = 'com.mxtech.videoplayer.ad'
 APP_NETFLIX = 'com.netflix.ninja'
+APP_NLZIET = 'nl.nlziet'
+APP_NOS = 'nl.nos.app'
+APP_NPO = 'nl.uitzendinggemist'
 APP_OCS = 'com.orange.ocsgo'
 APP_PLAY_GAMES = 'com.google.android.games'
 APP_PLAY_MUSIC = 'com.google.android.music'
 APP_PLAY_STORE = 'com.android.vending'
 APP_PLAY_VIDEOS = 'com.android.videos'
 APP_PLEX = 'com.plexapp.android'
 APP_PRIME_VIDEO = 'com.amazon.amazonvideo.livingroom'
-APP_PRIME_VIDEO_2 = 'com.amazon.firebat'
+APP_PRIME_VIDEO_FIRETV = 'com.amazon.firebat'
 APP_SMART_YOUTUBE_TV = 'com.liskovsoft.videomanager'
 APP_SPORT1 = 'de.sport1.firetv.video'
 APP_SPOTIFY = 'com.spotify.tv.android'
 APP_STEAM_LINK = 'com.valvesoftware.steamlink'
 APP_SYFY = 'com.amazon.webapps.nbc.syfy'
+APP_T2 = 'tv.perception.clients.tv.android'
+APP_TED = 'com.ted.android.tv'
+APP_TUNEIN = 'tunein.player'
 APP_TVHEADEND = 'de.cyberdream.dreamepg.tvh.tv.player'
-APP_TWITCH = 'tv.twitch.android.viewer'
-APP_TWITCH_APP = 'tv.twitch.android.app'
+APP_TWITCH = 'tv.twitch.android.app'
+APP_TWITCH_FIRETV = 'tv.twitch.android.viewer'
+APP_VEVO = 'com.vevo.tv'
 APP_VH1 = 'com.mtvn.vh1android'
+APP_VIMEO = 'com.vimeo.android.videoapp'
 APP_VLC = 'org.videolan.vlc'
+APP_VOYO = 'com.phonegap.voyo'
 APP_VRV = 'com.ellation.vrv'
 APP_WAIPU_TV = 'de.exaring.waipu.firetv.live'
 APP_WATCH_TNT = 'com.turner.tnt.android.networkapp'
 APP_YOUTUBE = 'com.google.android.youtube.tv'
-APP_YOUTUBE_2 = 'com.amazon.firetv.youtube'
+APP_YOUTUBE_FIRETV = 'com.amazon.firetv.youtube'
+APP_YOUTUBE_KIDS = 'com.google.android.youtube.tvkids'
+APP_YOUTUBE_KIDS_FIRETV = 'com.amazon.firetv.youtube.kids'
+APP_YOUTUBE_MUSIC = 'com.google.android.youtube.tvmusic'
 APP_YOUTUBE_TV = 'com.google.android.youtube.tvunplugged'
 APP_ZIGGO_GO_TV = 'com.ziggo.tv'
 APPS = {APP_AE_TV: 'A&E',
+        APP_AMAZON_PRIME_VIDEO: 'Amazon Prime Video',
```

Comment:
Renamed `APP_AMZ_VIDEO` as per discussion https://github.com/JeffLIrion/python-androidtv/pull/199#issuecomment-706712193.

Simply used the name associated to the application identifier on Google Play ([com.amazon.avod.thirdpartyclient](https://play.google.com/store/apps/details?id=com.amazon.avod.thirdpartyclient)).